socorro/processor/processor_pipeline.py

```diff
@@ -58,87 +58,32 @@
     UserDataRule,
 )
 
-DEFAULT_RULES = [
-    # fix the raw crash removing null characters
-    DeNullRule,
-    # rules to change the internals of the raw crash
-    ProductRewrite,
-    ESRVersionRewrite,
-    PluginContentURL,
-    PluginUserComment,
-    # rules to transform a raw crash into a processed crash
-    IdentifierRule,
-    MinidumpSha256Rule,
-    BreakpadStackwalkerRule2015,
-    ProductRule,
-    UserDataRule,
-    EnvironmentRule,
-    PluginRule,
-    AddonsRule,
-    DatesAndTimesRule,
-    OutOfMemoryBinaryRule,
-    JavaProcessRule,
-    MozCrashReasonRule,
-    # post processing of the processed crash
-    CrashingThreadRule,
-    CPUInfoRule,
-    OSInfoRule,
-    BetaVersionRule,
-    ExploitablityRule,
-    FlashVersionRule,
-    OSPrettyVersionRule,
-    TopMostFilesRule,
-    ThemePrettyNameRule,
-    MemoryReportExtraction,
-    # a set of classifiers to help with jit crashes
-    JitCrashCategorizeRule,
-    # generate signature now that we've done all the processing it depends on
-    SignatureGeneratorRule,
-]
```

Comment:
The `ProcessorPipeline` owns configuration for and instantiating these, so I figure it should own the list as well.